content/_includes/page/meta.njk

```diff
@@ -37,7 +37,7 @@
         {{ post.pubdate(
           start=page.date,
           end=end,
-          format='range' if oss or client else 'long'
+          format='range' if oss or client or event else 'long'
```

Comment:
I added this `event` data to talks/workshops so that we could replace the specific date with `since` -- but feel free to adjust if there's a better way to do this.